scripts/install.sh

```diff
@@ -7,18 +8,30 @@ ubuntu_default_editor="atril"
 
 read -p "Are you sure you want to install gnu-koryavov on yout system (Y/N): " ans
 
-echo "Determining OS version..."
-os_info=$(cat /etc/*release)
-echo "$os_info"
 
-is_ubuntu=$(echo $os_info | grep -i "ubuntu")
+editors=(okular atril evince xreader zathura)
+editor="NO EDITOR"
 
-if [[ $? -eq 0 ]] ; then
+for item in ${editors[*]}; do
 
-    echo "Running on ubuntu machine. Default editor: $ubuntu_default_editor"
+    $item --version &> /dev/null
+    if [[ $? == 0 ]]; then
+        editor=$item
+        break
+    fi
 
+done
+
+if [[ $editor == "NO EDITOR" ]]; then
+
+    echo "No supported DjVU viewer intalled"
+    exit 0
 fi
 
+echo "Choosen editor: $editor. You can change it as wrote in README"
+sed -i "s/okular.sh/$editor.sh/g" ./../config.conf
```

Comment:
To my mind it's better to use something like `sed -i '/pdfviewer_script/s/=.*\.sh/=$editor/' config.conf`

Reply:
Yes, I agree
We use DjVU, btw

Reply:
I fixed it

Reply:
I think specifying `~/gnu-koryavov` makes this part of the script less flexible

Reply:
Yes, but if we use without specifying, then it will replace all after '='-symbol.

Reply:
That's what we're trying to do, aren't we?

Reply:
We trying to replace only name of script, but not all path to it

Reply:
Yeah, but I meant this `sed -i "/djvuviewer_script/s/=.*\.sh/=~\/gnu-koryavov\/$editor.sh/" ./../config.conf` is more flexible than this `sed -i "/djvuviewer_script/s/=~\/gnu-koryavov\/.*\.sh/=~\/gnu-koryavov\/$editor.sh/" ./../config.conf` because it replaces the whole parameter's value with the script path needed

Reply:
Ok, I agree. I already did.